src/rpc/AppRpcProvider.ts

```diff
@@ -113,10 +113,9 @@ export default class AppRpcProvider extends AppStaticJsonRpcProvider {
       for (const { provider, performance } of this.providerEvaluations) {
         performance.callCount++
         try {
-          return await provider.perform(method, params)
+          return useSend ? await provider.send(method, params as Array<any>) : await provider.perform(method, params)
```

Comment:
Did you figure out what `send` vs `perform` means?

Reply:
send sends the arguments as raw as they are defined by caller.
with perform you can name the variables, eg:
`return [ "eth_getStorageAt", [ getLowerCase(params.address), hexZeroPad(params.position, 32), params.blockTag ] ];`

I don't like this idea, as its all badly typed (all those params. values are any's), but I didn't want to start to refactor it